packages/react-composites/tests/browser/chat/fake-adapter/FileSharing.test.ts

```diff
@@ -254,48 +225,47 @@ test.describe('Filesharing Message Thread', async () => {
             uploadComplete: true
           }
         ])
-      }
-    };
-    const otherUsersArgs = users.slice(1).map((user) => ({
-      user,
-      qArgs: {
-        useFileSharing: 'true'
-      }
-    }));
-    const usersWithArgs = [firstUserArgs, ...otherUsersArgs];
-    await chatTestSetupWithPerUserArgs({
-      pages,
-      usersWithArgs,
-      serverUrl
-    });
-  });
-  test('contains File Download Card', async ({ pages }) => {
+      })
+    );
     const testMessageText = 'Hello!';
-    const page0 = pages[0];
 
-    await waitForChatCompositeToLoad(page0);
-    await sendMessage(page0, testMessageText);
-    await waitForMessageDelivered(page0);
-    await page0.waitForSelector(dataUiId('file-download-card-group'));
+    await waitForChatCompositeToLoad(page);
+    await sendMessage(page, testMessageText);
+    await waitForMessageDelivered(page);
+    await page.waitForSelector(dataUiId('file-download-card-group'));
 
-    await stubMessageTimestamps(page0);
-    expect(await page0.screenshot()).toMatchSnapshot('filesharing-file-download-card-in-sent-messages.png');
+    await stubMessageTimestamps(page);
+    expect(await page.screenshot()).toMatchSnapshot('filesharing-file-download-card-in-sent-messages.png');
+  });
 
-    const page1 = pages[1];
-    await waitForTypingIndicatorHidden(page1);
+  test('contains File Download Card in remote message', async ({ serverUrl, page }) => {
```

Comment:
I had to create a separate test here with qArg `hasRemoteFileSharingMessage` to get filesharing message from a remote participant to get snapshot 'filesharing-file-download-card-in-received-messages.png'